app/models/account.rb

```diff
@@ -88,10 +103,14 @@ def self.for_facility(facility)
     accounts = scoped
 
     if facility.single_facility?
-      accounts = accounts.where("accounts.type in (:allow_all) or (accounts.type in (:limit_one) and accounts.facility_id = :facility)",
-            {:allow_all => AccountManager::GLOBAL_ACCOUNT_CLASSES,
-              :limit_one => AccountManager::FACILITY_ACCOUNT_CLASSES,
-              :facility => facility})
+      accounts = accounts.where(
+        "accounts.type in (:allow_all) or (accounts.type in (:limit_one) and accounts.facility_id = :facility)",
+        {
```

Comment:
While we're cleaning up the syntax, the curly braces should be able to go away, too.